guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/repository/wizard/pages/RepositoryInfoPageTest.java

```diff
@@ -161,6 +173,7 @@ public void testInvalidRepositoryNameChange( ) {
 
         verify( view, times( 2 ) ).getName();
         verify( view, times( 1 ) ).setNameErrorMessage( anyString() );
+        verify( statusChangeHandler, never() ).handleEvent( any( WizardPageStatusChangeEvent.class ) );
```

Comment:
Same here: Does the ```WizardPageStatusChangeEvent``` not get called whenever something changes on the page? I'd have thought selecting an invalid repository name would still trigger a status event to be triggered?